lib/tessel/deploy.js

```diff
@@ -54,6 +54,26 @@ Tessel.prototype.deployScript = function(opts) {
   });
 };
 
+Tessel.prototype.restartScript = function(opts) {
+  var self = this;
+  var isPush = opts.type === 'flash';
+  var filepath = isPush ? PUSH_PATH : RUN_PATH;
+
+  return new Promise(function(resolve, reject) {
+    actions.findProject(opts.entryPoint).then(function(project) {
```

Comment:
Those errors should just be forwarded through as rejections—right? Right now it looks like USBProcess is just emitting an error event and nothing is handling that. I'm not sure this feature is the right place to fix that.